build.gradle.kts

```diff
@@ -7,3 +15,42 @@ repositories {
    google()
    gradlePluginPortal() // tvOS builds need to be able to fetch a kotlin gradle plugin
 }
+
+catalog {
+   val groupId = project.group.toString()
+   val extensionsGroupId = "$groupId.extensions"
+
+   println("using groupId: $groupId and version: $publishVersion for version catalog")
+
+   versionCatalog {
+      listOf(
+         "assertions-api",
```

Comment:
Is there any way to generate this list automatically? From inspecting subprojects perhaps?